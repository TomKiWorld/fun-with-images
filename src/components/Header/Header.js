import React from 'react';
import Navigation from '../Navigation/Navigation';
import Logo from '../Logo/Logo';

/**
 * Site header 
 * 
 * Required props:
 * - isSignedIn => In order to dispaly the correct naviagtion from state
 * - onRouteChange => Function to change route state
 * - route => The current route from state
 */
const Header = ({isSignedIn, onRouteChange, route}) => {
  return (
    <header className='site-header'>
      <Navigation 
          isSignedIn={isSignedIn}
          onRouteChange={onRouteChange}
          route={route} />
      <Logo />
    </header>
  );
}

export default Header;
