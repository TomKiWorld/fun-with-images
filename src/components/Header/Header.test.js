import { shallow } from 'enzyme';
import React from 'react';
import Header from './Header';

describe('Header Component', () => {
  it('Expect to render Header Component', () => {
    expect(shallow(<Header />)).toMatchSnapshot();
  });
});