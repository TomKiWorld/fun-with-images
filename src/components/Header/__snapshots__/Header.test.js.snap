// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Expect to render Header Component 1`] = `ShallowWrapper {}`;
